### DISCUSSION TITLE
how-do-i-build-my-blog

### DISCUSSION THREAD

**Discussion by giscus** [bowencool/blog#77] [Announcements]
# how-do-i-build-my-blog

本文讲述了作者在搭建博客的一些思考、权衡以及实践过程中的经验。

https://blog.bowen.cool/zh/posts/how-do-i-build-my-blog

**Comment by LYZ10** [comment]
您好呀，想请问下您方便也分享一个面相技术小白的博客搭建教程吗？谢谢。

**Reply by bowencool** [reply]
不想折腾的可以直接用 Notion、飞书文档之类的，编辑页面可以直接发布，不需要任何代码及命令。想折腾的话用 Hexo，Hexo 教程太多了，我不会再重复了。

**Comment by LYZ10** [comment]
好的哈，谢谢您的回复。还是想折腾下的，notion毕竟是商业平台。
&nbsp;


ECG
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年5月24日(星期五) 中午1:31
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [bowencool/blog] how-do-i-build-my-blog (Discussion #77)














 
想折腾的话用 Hexo，不想折腾的可以直接用 Notion，Notion 直接可以发布，不需要任何代码及命令。

**Comment by eerison** [comment]
I liked you reaction button 👏👏👏

Really detailed pos, specially the language feature, I will take a look again later.

Thank you 😊
